### PR TITLE
[10.0][IMP] account_financial_report_qweb: color scheme

### PR DESCRIPTION
**Pull request by i-vyshnevska**
Change backgroud to blank on financial reports

### CODE REVIEW

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by JordiBForgeFlow** [comment]
In Odoo CE, when I check without your changes:

image


And with your PR:

image

Seems better not to apply the default white color, right?

**Comment by pedrobaeza** [comment]
:-1: I prefer previous one.

**Comment by JordiBForgeFlow** [comment]
I agree. @i-vyshnevska any opinion?

**Comment by fclementic2c** [comment]
@jbeficent @pedrobaeza 
Check it on the enterprise version too:
screenshot-beta-erp bsonetwork net-2019 01 10-13-31-32

Can't we have a transparent background so it fit both community & enterprise versions ?

**Comment by pedrobaeza** [comment]
@fclementic2c it's not possible to check it in the enterprise version, and this doesn't target enterprise users any way. For that enterprise lovers, it's supposed that enterprise account reports are "better" than this...

**Comment by fclementic2c** [comment]
@pedrobaeza you're too fast. see my updated comment.

...and no, OCA financial reports are far better than enterprise reports. Yesterday again I had the confirmation from my bigger customer at the moment in Paris. There is no way the enteprise report can be acceptable for a bigger company.

**Comment by pedrobaeza** [comment]
I see no bigger improvement putting the initial blank. Any way, that kind of change should affect only enterprise, no community. And if not, switch also to OCA responsive. In v12, we have made a fine work and I consider now it's also better than enterprise one: https://github.com/OCA/web/pull/1066

**Comment by fclementic2c** [comment]
@i-vyshnevska Can we do the background change only if we install the module on a enterprise version ?

**Comment by i-vyshnevska** [comment]
@fclementic2c need to check

**Comment by i-vyshnevska** [comment]
@jbeficent  it's better now
screenshot

**Review by fclementic2c — APPROVED** [review]

File: account_financial_report_qweb/static/src/css/report.css [hunk 1]
```diff
@@ -96,11 +97,12 @@
...
 .o_account_financial_reports_page {
-    background-color: @odoo-view-background-color;
+    background-color: white;
     color: @odoo-main-text-color;
     padding-top: 10px;
     width: 90%;
     margin-right: auto;
     margin-left: auto;
     font-family: Helvetica, Arial;
+    padding: 40px
```
Comment (pedrobaeza) [inline account_financial_report_qweb/static/src/css/report.css]: Why changing padding?

File: account_financial_report_qweb/static/src/css/report.css [hunk 2]
```diff
@@ -96,11 +97,12 @@
 }
 
 .o_account_financial_reports_page {
-    background-color: @odoo-view-background-color;
+    background-color: white;
```
Comment (pedrobaeza) [inline account_financial_report_qweb/static/src/css/report.css]: Isn't a variable for enterprise background? I still prefer to keep "transparent" background

File: account_financial_report_qweb/static/src/css/report.css [hunk 3]
```diff
@@ -65,6 +65,7 @@
 .act_as_cell.amount {
     word-wrap:normal;
     text-align:right;
+    padding: 2px;
```
Comment (pedrobaeza) [inline account_financial_report_qweb/static/src/css/report.css]: Why this extra padding? This makes bigger full table.

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: account_financial_report_qweb/static/src/css/report.css (hunk 1 above)
Comment (i-vyshnevska) [reply account_financial_report_qweb/static/src/css/report.css]: so this looks better https://github.com/OCA/account-financial-reporting/pull/501#issuecomment-453060324

Comment (pedrobaeza) [reply account_financial_report_qweb/static/src/css/report.css]: Background color has nothing to do with padding.

File: account_financial_report_qweb/static/src/css/report.css (hunk 2 above)
Comment (simahawk) [reply account_financial_report_qweb/static/src/css/report.css]: this is broken anyway, is not LESS ;)

File: account_financial_report_qweb/static/src/css/report.css (hunk 1 above)
Comment (simahawk) [reply account_financial_report_qweb/static/src/css/report.css]: @i-vyshnevska pls post the difference w/ and w/out.
The issue is: it's really ugly w/out padding as you get the dark gray BG in touch w/ the borders and very next to the text.

File: account_financial_report_qweb/static/src/css/report.css (hunk 2 above)
Comment (simahawk) [reply account_financial_report_qweb/static/src/css/report.css]: we should move the whole file to LESS and then play w/ the variables

File: account_financial_report_qweb/static/src/css/report.css (hunk 1 above)
Comment (pedrobaeza) [reply account_financial_report_qweb/static/src/css/report.css]: Take into account that this also affects PDF report, where a padding would affect to have 200 pages, or 400...

Comment (i-vyshnevska) [reply account_financial_report_qweb/static/src/css/report.css]: @pedrobaeza  this padding not depend of amount of account entries if you have a lot of pages it add padding only in the end 
image

File: account_financial_report_qweb/static/src/css/report.css (hunk 3 above)
Comment (i-vyshnevska) [reply account_financial_report_qweb/static/src/css/report.css]: w/ padding 
image

without 
image

maybe decrease it 1px would be optimal variant

File: account_financial_report_qweb/static/src/css/report.css (hunk 2 above)
Comment (i-vyshnevska) [reply account_financial_report_qweb/static/src/css/report.css]: @pedrobaeza  what do we agreed? keep transparent for community and white for enterprise?
to do that, I don't see other option, only a creating separate module just for changing background, in a way that we want

Comment (pedrobaeza) [reply account_financial_report_qweb/static/src/css/report.css]: My question was:

> Isn't a variable for enterprise background?

and my purpose is to let it transparent, but in both versions.

File: account_financial_report_qweb/static/src/css/report.css (hunk 3 above)
Comment (pedrobaeza) [reply account_financial_report_qweb/static/src/css/report.css]: Well, why wasting that precious pixels in not so wide screens? I don't see any advantage.

Comment (i-vyshnevska) [reply account_financial_report_qweb/static/src/css/report.css]: it probably not about wide, cos there a lot of free space and padding left, right have no influence on columns width,
 maybe only  on mobile devices, but for mobile there problems and without  padding 
and looks fine for screens from 1024px 
image

it effects on height of report  but easier to read, open some item

Comment (pedrobaeza) [reply account_financial_report_qweb/static/src/css/report.css]: I prefer to remove all the paddings.

Comment (i-vyshnevska) [reply account_financial_report_qweb/static/src/css/report.css]: done

File: account_financial_report_qweb/static/src/less/report.less [hunk 4]
```diff
@@ -103,4 +103,5 @@
     margin-right: auto;
     margin-left: auto;
     font-family: Helvetica, Arial;
+    padding: 20px;
```
Comment (pedrobaeza) [inline account_financial_report_qweb/static/src/less/report.less:106]: But this padding is still changed...

**Comment by pedrobaeza** [comment]
And what has happened with background?

File: account_financial_report_qweb/static/src/less/report.less (hunk 4 above)
Comment (i-vyshnevska) [reply account_financial_report_qweb/static/src/less/report.less:106]: because w/ less file background compiled to white color, and this padding we need

Comment (pedrobaeza) [reply account_financial_report_qweb/static/src/less/report.less:106]: @jbeficent what do you think about these changes? I still don't like the white background and the unneeded padding

**Review by simahawk — APPROVED** [review]

File: account_financial_report_qweb/static/src/css/report.css (hunk 2 above)
Comment (fclementic2c) [reply account_financial_report_qweb/static/src/css/report.css]: @pedrobaeza but transparent in enterprise version means grey and I show you above that this is not very much readable. Do I understand properly ?

Comment (i-vyshnevska) [reply account_financial_report_qweb/static/src/css/report.css]: @fclementic2c  it fixed and @odoo-view-background-color will give a white color in report

**Comment by fclementic2c** [comment]
@pedrobaeza And if we just set the background to white only on the line where the account name appear ? because in the end all I need is to be able to read the account (black on grey is not ideal)

**Comment by pedrobaeza** [comment]
I let the final word to @jbeficent

**Review by leemannd — APPROVED** [review]
Working.
Tested only with enterprise license.

**Comment by fclementic2c** [comment]
I close it

**Comment by fclementic2c** [comment]
@simahawk says it is not usefull to close it so... I reopen it

**Comment by i-vyshnevska** [comment]
@jbeficent @pedrobaeza passby this PR or close it 
pls consider that it has fixes as less variables file are not reachable

**Pull request closed without merging**